hack/lib/build/version.sh

```diff
@@ -40,9 +40,10 @@ function os::build::version::openshift_vars() {
 			# Try to match the "git describe" output to a regex to try to extract
 			# the "major" and "minor" versions and whether this is the exact tagged
 			# version or whether the tree is between two tagged versions.
-			if [[ "${OS_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)(\.[0-9]+)*([-].*)?$ ]]; then
+			if [[ "${OS_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)\.([0-9]+)*([-].*)?$ ]]; then
```

Comment:
This was also matching `ose` version e.g. `v3.5.6.7` so I'd add another `\.([0-9]+)` stanza for the patch but leave the trailing `(\.[0-9]+)*` alone